packages/hdwallet-ledger-webusb/src/transport.ts

```diff
@@ -30,6 +76,15 @@ export class LedgerWebUsbTransport extends LedgerTransport {
     return await getDeviceInfo(this.transport)
   }
 
+  public async open() {
+    const ledgerTransport = await getTransport()
+    this.transport = ledgerTransport
```

Comment:
```suggestion
    this.transport = await getTransport()
```

Reply:
haha, yeah I kept thinking I needed to return the promise first. Thanks!